builder/openstack/step_run_source_server.go

```diff
@@ -37,6 +40,19 @@ func (s *StepRunSourceServer) Run(state multistep.StateBag) multistep.StepAction
 		networks[i].Uuid = networkUuid
 	}
 
+	encodedUserData := ""
+	if s.UserDataFile != "" {
+		data, err := ioutil.ReadFile(s.UserDataFile)
```

Comment:
Instead of reading the entire file into memory here, and then converting the string contents into base64 below, can we stream the file into a base64 encoder instead?